source/Body.h

```diff
@@ -78,7 +78,9 @@ class Body {
 	// Set the color swizzle.
 	void SetSwizzle(int swizzle);
 
-	double Alpha() const;
+	double Alpha(const Point &drawCenter) const;
```

Comment:
```suggestion
	// Functions determining the current alpha value of the body
	// dependent on the position of the body to the center of screen.
	double Alpha(const Point &drawCenter) const;
```
?

Reply:
~~ur missing a comma at the end of that line I think~~

Reply:
```suggestion
	// Functions determining the current alpha value of the body,
	// dependent on the position of the body to the center of screen.
	double Alpha(const Point &drawCenter) const;
```
correct now? my punctuation is terrible